Styles/portfolio.css

```
@@ -0,0 +1,77 @@
+body, htmls{
```

I think you meant `html`:

```suggestion
body, html {
```